lax/lichens/sciencerun0.py

```diff
@@ -56,7 +56,21 @@ class S1LowEnergyRange(RangeLichen):
     allowed_range = (0, 200)
     variable = 'cs1'
 
+class S1MaxPMT(Lichen):
+    """ S1Max PMT Cut.
```

Comment:
Can you make this short description more understandable?  Cut on maximum contribution of single PMT or something